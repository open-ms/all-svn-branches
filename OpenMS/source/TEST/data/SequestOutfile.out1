
 example-rabbit.1055.1055.3.out
 TurboSEQUEST v.27 (rev. 12), (c) 1999-2003
 Molecular Biotechnology, Univ. of Washington, J.Eng/S.Morgan/J.Yates
 Licensed to ThermoFinnigan Corp.
 03/16/2004, 02:32 PM, 4.6 sec. on MILES
 (M+H)+ mass = 1966.7911 ~ 2.5000 (+3), fragment tol = 0.0000, MONO/MONO
 total inten = 7275.7, lowest Sp = 225.4, # matched peptides = 74886
 # amino acids = 1231264, # proteins = 4836, D:/database/rabbit.fasta
 ion series nABY ABCDVWXYZ: 0 1 1 0.0 1.0 0.0 0.0 0.0 0.0 0.0 1.0 0.0
 display top 12/12, ion % = 0.0, CODE = 101010
 (M* +15.995) C=160.038 Enzyme:None selected
 
  #   Rank/Sp      Id#     (M+H)+   deltCn   XCorr    Sp    Sf    P    Ions  Reference                            Peptide
 ---  --------  --------  --------  ------  ------   ----  ----  ----  ----  ---------                            -------
  1.   1 / 80          0 1967.0013  0.0000  1.5789   310.3 0.05    0 18/64  gi|544379|sp|P35574|GDE_RABIT   +2
