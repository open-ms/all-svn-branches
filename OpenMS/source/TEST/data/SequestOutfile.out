
 example-rabbit.1055.1055.3.out
 TurboSEQUEST v.27 (rev. 12), (c) 1999-2003
 Molecular Biotechnology, Univ. of Washington, J.Eng/S.Morgan/J.Yates
 Licensed to ThermoFinnigan Corp.
 03/16/2004, 02:32 PM, 4.6 sec. on MILES
 (M+H)+ mass = 1966.7911 ~ 2.5000 (+3), fragment tol = 0.0000, MONO/MONO
 total inten = 7275.7, lowest Sp = 225.4, # matched peptides = 74886
 # amino acids = 1231264, # proteins = 4836, D:/database/rabbit.fasta
 ion series nABY ABCDVWXYZ: 0 1 1 0.0 1.0 0.0 0.0 0.0 0.0 0.0 1.0 0.0
 display top 12/12, ion % = 0.0, CODE = 101010
 (M* +15.995) C=160.038 Enzyme:None selected
 
  #   Rank/Sp      Id#     (M+H)+   deltCn   XCorr    Sp    Sf    P    Ions  Reference                            Peptide
 ---  --------  --------  --------  ------  ------   ----  ----  ----  ----  ---------                            -------
  1.   1 / 80          0 1967.0013  0.0000  1.5789   310.3 0.05    0 18/64  gi|544379|sp|P35574|GDE_RABIT   +2   C.ETQAWSIATILETLYDL.-
                       0  gi|2136928|pir||S38758 amylo-1,6-g
                       0  gi|294478 (L10605) amylo-1, 6-gluc
  2.   2 / 85          0 1967.1985  0.0390  1.5173   308.3 0.04    0 19/64  gi|1172436|sp|P46029|PET2_RABIT +1   M.QVLNPLLVLIFIPLFDL.V
                       0  gi|975890 (U32507) oligopeptide tr
  3.   3 /117          0 1968.1244  0.0501  1.4998   292.4 0.02    0 17/72  gi|1430907|gnl|PID|e148876           L.WVELGPSVLAGVGVM*VLLI.P
  4.   4 /  1          0 1964.9275  0.0627  1.4799   530.9 0.14    0 24/72  gi|114314|sp|P20647|ATCE_RABIT  +7   S.FDEITAMTGDGVNDAPALK.K
                       0  gi|114310|sp|P04192|ATCD_RABIT CAL
                       0  gi|67962|pir||PWRBMC Ca2+-transpor
                       0  gi|67961|pir||PWRBSC Ca2+-transpor
                       0  gi|109166|pir||A33881 Ca2+-transpo
                       0  gi|224621|prf||1109242A ATPase,Ca
                       0  gi|1469 (X02814) Ca2+-ATPase [Oryc
                       0  gi|164739 (J04703) calcium-ATPase
